core-libs/setup/ssr/providers/server-app-dist-folder.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { inject, InjectionToken } from '@angular/core';
+import { REQUEST } from '@nguniversal/express-engine/tokens';
+
+/**
+ * The absolute directory on the server pointing to the `/dist` folder of the application.
+ *
+ * It can be used for example to determine the path to the application's assets on the server,
+ * so to load them directly from the file system (e.g. using Node's `fs` package).
+ */
+export const SERVER_APP_DIST_FOLDER = new InjectionToken<string>(
+  'BROWSER_APP_DIST_FOLDER',
```

Comment:
```suggestion
  'SERVER_APP_DIST_FOLDER',
```
Shouldn't it be `SERVER_APP_DIST_FOLDER`?

Reply:
Btw, it's not used anywhere atm. I guess it's because this is a draft PR.

Reply:
Yea, it can be a left over after my previous implementation that was using `fs` and had to know the project's absolute dist folder path in the system.